### DISCUSSION TITLE
Fuzzel 1.10.* run window size

### DISCUSSION THREAD

**Discussion by nebulosa2007** [YaLTeR/niri#288] [General]
I don't know if this is a Niri issue or not. 

After updating fuzzel to 1.10.1 from 1.9.2, the launch window opens in full screen mode:

![Screenshot from 2024-04-13 13-54-24](https://github.com/YaLTeR/niri/assets/85841412/23ab3f2e-53ce-4488-bd39-92d34ba0e41b)
1.9.2

![Screenshot from 2024-04-13 14-02-45](https://github.com/YaLTeR/niri/assets/85841412/e7639700-59ad-40b3-8c55-bec3f35ab628)
1.10.1

Any thoughts?
```console
$ niri -V
niri 0.1.4 (7ff2de1)
```

Tried to delete the fuzzel config file - didn't help.

**Comment by YaLTeR** [comment]
Seems like it could be a Smithay layer-shell positioning issue, because this reproduces in Anvil and does not reproduce in Sway.

**Comment by nebulosa2007** [comment]
After upgrading `fuzzel` 1.10.1 -> 1.10.2 [issue is fixed.](https://codeberg.org/dnkl/fuzzel/pulls/332)
